presentation/chart_view.py

```diff
@@ -0,0 +1,63 @@
+import urllib.parse as urlparse
+from datetime import datetime, timedelta, timezone
+from typing import List, Tuple
+from urllib.parse import parse_qs, unquote
+
+import dash_core_components as dcc
+import dash_html_components as html
+import plotly.graph_objs as go
+
+from domain.agents import Agents
+from domain.metric import Metric
+from domain.model import MeshResults
+from domain.model.mesh_results import HealthItem, MeshResults
+from domain.types import AgentID
+from presentation.health import Health
+
+
+class ChartView:
+    @staticmethod
+    def make_layout(from_agent, to_agent: AgentID, metric: Metric, results: MeshResults) -> html.Div:
+        # make chart title
+        agents = Agents(results)
+        from_alias = agents.get_alias(from_agent)
+        to_alias = agents.get_alias(to_agent)
+        title = f"{metric.value}: {from_alias} -> {to_alias}"
+
+        # make chart data
+        health = Health(from_agent, to_agent, results)
+        xdata = [ChartView._utc_to_localtime(item.time) for item in health.items]
```

Comment:
`utc_to_localtime` should be helper function as opposed to a "private" static method of `ChartView`. It is not specific to `ChartView` and it may come handy at other places too.